conference.js

```diff
@@ -157,13 +166,13 @@ function createInitialLocalTracksAndConnect(roomName) {
     return Promise.all([ tryCreateLocalTracks, connect(roomName) ])
         .then(([tracks, con]) => {
             APP.store.dispatch(mediaPermissionPromptVisibilityChanged(false));
-            if (audioAndVideoError) {
+            if (audioAndVideoError || noVideo) {
```

Comment:
Hmm but isn't it true that you will generate only audio errors here because video is not requested at all? Sorry if I'm missing something!